recipes/esmlab-regrid/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "esmlab-regrid" %}
+{% set version = "2019.5.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1a6492e1f19d52b8631b6e38809ea67a2005ca7b9ca9e56c64022541927f444d
+  
+
+build: 
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py2k or win]
```

The package seems to be a pure Python package, so this could be `noarch`
```suggestion
  noarch: python
```

---

@xhochy, thank you for your feedback. That's right. The package is a pure Python package but one of its dependency, namely [esmpy](https://anaconda.org/conda-forge/esmpy) is not supported on windows. How should I skip building for Windows?